test/check-application

```diff
@@ -28,6 +28,22 @@ IMG_BUSYBOX_LATEST = IMG_BUSYBOX + ":latest"
 IMG_REGISTRY = "localhost/test-registry"
 IMG_REGISTRY_LATEST = IMG_REGISTRY + ":latest"
 
+NGINX_DEFAULT_CONF = """
+server {
+  listen 80;
+  listen  [::]:80;
+  server_name localhost;
+
+  location / {
+    proxy_pass http://localhost:5000/;
+  }
+
+  location ^~ /v2/_catalog {
```

Comment:
Please add a comment for our future selves: Something like "Simulate a registry without search API, like ghcr.io"